resources/js/Components/Table/CharacterDetail/CharacterDetailTable.tsx

```diff
@@ -154,27 +157,17 @@ const CharacterDetailTable = ({rows}: CharacterDetailTablePropType) => {
                                             component='th'
                                             id={labelId}
                                             scope='row'>
-                                            {row.date}
-                                        </StyledTableCell>
-                                        <StyledTableCell>
-                                            {row.adventure}
-                                        </StyledTableCell>
-                                        <StyledTableCell>
-                                            {row.session}
-                                        </StyledTableCell>
-                                        <StyledTableCell>
-                                            {row.level}
-                                        </StyledTableCell>
-                                        <StyledTableCell>
-                                            {row.gp}
+                                            {row.date_played}
                                         </StyledTableCell>
                                         <StyledTableCell>
-                                            {row.downtime}
+                                            {row.adventure.title}
                                         </StyledTableCell>
+                                        <StyledTableCell>{row.session}</StyledTableCell>
+                                        <StyledTableCell>{row.levels}</StyledTableCell>
+                                        <StyledTableCell>{row.gp}</StyledTableCell>
+                                        <StyledTableCell>{row.downtime}</StyledTableCell>
                                         <StyledTableCell>
-                                            <ColorButton variant='outlined'>
-                                                {row.magicItems}
-                                            </ColorButton>
+                                            <ColorButton>hi</ColorButton>
```

Comment:
not sure what this is for

Reply:
magic items on the entry table looked like they were buttons so this is just a placeholder